test/functional/device/xctest-e2e-specs.js

```diff
@@ -17,11 +17,11 @@ if (process.env.LAUNCH_WITH_IDB) {
     let driver;
 
     beforeEach(async function () {
-      driver = await initSession({
```

Comment:
Actually, I haven't tested this local yet as I face an issue where idb_companion never gets ready. It look fine if I manually run `idb-companion --udid xxx` 🤔 
```
[IDB] Failed to start idb_companion: Was unable to acquire a GRPC port after 60 seconds timeout
```
If CI doesn't pass, can I take it out of scope for this PR for now and take a look later?

Reply:
I assume there is an issue with idb itself. Integration tests of appium-idb mofule fail with the same error. Feel free to investigate that when you have time

Reply:
> If CI doesn't pass either, can I take it out of scope for this PR for now and take a look later?

Sure, this is completely unrelated topic. Let me know if you need help there

Reply:
@rerorero I would also be happy to answer all you dev-related question in Slack. Share your email if you'd like to be invited there.

Reply:
That's very cool, thank you!  My email address is `nato9598@hotmail.co.jp`

I disabled this test for now.
https://github.com/appium/appium-xcuitest-driver/pull/1461/commits/ca5f1260868a81100f74193f4a89e1d383e9ce45

Reply:
Thank you for the invitation, I've joined the channel.